Source/WebKit/UIProcess/API/Swift/WebPage.swift

```diff
@@ -102,6 +102,15 @@ extension WebPageWebView {
         get { scrollView.bouncesHorizontally }
         set { scrollView.bouncesHorizontally = newValue }
     }
+
+    @_spi(CrossImportOverlay)
+    public var allowsMagnification: Bool {
```

Comment:
Doesn't this need a macOS implementation?

Reply:
nope, because macOS WKWebView already has allowsMagnification allowsMagnification